.github/workflows/ci2.yml

```diff
@@ -62,33 +62,21 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        distro: [ubuntu.20.04, centos.8, leap.15, fedora, rocky, alma]
+        distro: [centos.8, leap.15, fedora]
         compiler: [clang, gcc]
         include:
           - distro: fedora
             base: centos.8
             with: fedora:34
-            java: NO
-          - distro: rocky
-            base: centos.8
-            with: rockylinux/rockylinux:8
-            java: YES
-          - distro: alma
-            base: centos.8
-            with: almalinux:8
-            java: YES
+            java: no
           - distro: centos.8
             base: centos.8
             with: centos:8
```

Comment:
Let's drop centos.8 here and use Rocky Linux instead.

Reply:
Done, testing it now.  I ordered the matrix entries at the same time.